WordPress/Classes/ViewRelated/Blog/Blog Dashboard/BlogDashboardViewController.swift

```diff
@@ -85,15 +83,37 @@ final class BlogDashboardViewController: UIViewController {
 extension BlogDashboardViewController {
 
     private func createLayout() -> UICollectionViewLayout {
-        let layout = UICollectionViewCompositionalLayout {
+        UICollectionViewCompositionalLayout {
             (sectionIndex: Int, layoutEnvironment: NSCollectionLayoutEnvironment) -> NSCollectionLayoutSection? in
 
-            return self.createQuickLinksSection()
+            if sectionIndex == 0 {
```

Comment:
Relying on numbers might be tricky and error-prone. I think if we rely on the item itself this will be safier:

```suggestion
            if self.viewModel.dataSource?.itemIdentifier(for: IndexPath(row: 0, section: sectionIndex))?.id == .quickActions {
```

Of course, we can extract that to a method that based on the sectionIndex returns the item identifier or something like that.

Also, you may want to use `[unowned self]` or `[weak self]` for this block.

Reply:
Fixed! 8a3da0e